modules/ROOT/pages/prepare-to-install-service-mesh.adoc

```diff
@@ -156,12 +223,11 @@ To install the manifest, run:
 +
 `istioctl manifest apply -f <manifest-file.yaml>`
 
-=== Install Istio 1.4.x
+=== Install and Configure Istio 1.4.x
```

Comment:
We need to delete this section as Istio 1.4 is not supported in 1.1

Reply:
Done! Thanks!